### PR TITLE
revamp the GM lounge

### PR DESCRIPTION
**Pull request by mekolat**
goes with https://github.com/themanaworld/tmwa-client-data/pull/261

### CODE REVIEW

File: world/map/npc/functions/announcements.txt [hunk 1]
```diff
@@ -148,7 +130,7 @@ L_StartBroadcast:
     set $@GMSA_STATUS, 1;
     set $@GMSA_NAME$, strcharinfo(0);
     set $@GMSA_CURRENT_TICK, 0;
-    initnpctimer;
+    initnpctimer "Stone Board";
```
Comment (wushin) [inline world/map/npc/functions/announcements.txt]: @mekolat say what?

**Comment by wushin** [comment]
@mekolat merge conflict

**Comment by wushin** [comment]
@mekolat merge conflict

**Comment by mekolat** [comment]
@wushin done

**Comment by wushin** [comment]
menu's need close options. you have to use the M+ method which is a bit counter to how we did NPCs.

**Comment by mekolat** [comment]
why do we need a close option if they can just click the close button? can you further explain? is the close packet handled at all? do we need to add a OnDialogCloseEvent ?

**Comment by wushin** [comment]
Well don't because M+ does, but not sure if TMWA handles that properly. Second it's a UI thing. As I can page through the menu items with arrow keys but have to Tab I think to close. If there is a key that closed dialogue windows it be much better. Still we would have to train the players to use the key. As I was saying earlier if you look at how all the menus are designed they include a L_Close of some sort.

**Comment by wushin** [comment]
Additional note, the close ability for M+ can get you when scripting as handing an reward doing a next then setting the increased quest var allows exploit. But meh, we should deal with the implicit close from M+ as it's a packet that can be forged anyways. Meaning we should make sure scripts accept close at any point if an player is attached.

**Comment by mekolat** [comment]
well, I've always taken this into account anyways since o11c told me I should NEVER wait for player interaction to do something (ie wait for a player to close the window to set a variable

**Pull request merged**